Example_User_Service.yaml

```diff
@@ -0,0 +1,1308 @@
+        description: The description of the customer accommodation booking
+    example:
+      transportationReservationId: "fd133724-3450-47c3-b4a4-f888dbf74e40"
+      description: "A courtesy loaner car has been reserved for Ms. Jones"
+  TransportationOption:
+    description: This indicates the option of transportation that the customer selected.
+    required:
+      - transportationOptionId
+      - description
+    properties:
+      transportationOptionId:
+        type: string
+        format: uuid
+        description: The identifier of the transportation option
+      description:
+        type: string
+    example: 
+      transportationOptionId: 779913c9-bb93-46f0-8860-9c893128c62c
+      description: Some description
+  TransportationOffer:
+    description: This is the offer to provide transportation.
+    required:
+      - transportationOfferId
+      - description
+      - cost
+    properties:
+      transportationOfferId:
+        type: string
+        format: uuid
+        description: The identifier of the transportation offer
+      description:
+        type: string
+        description: The description transportation offer
+      cost:
+        $ref: '#/definitions/Cost'
+    example:
+      transportationOfferId: "fd133724-3450-47c3-b4a4-f888dbf74e40"
+      description: "Midsized loaner car"
+      cost:
+        baseCost: 0.0
+        currencyType: "GBP"
+  Cost:
+    description: This is the description of the cost.
+    required:
+      - baseCost
+      - currencyType
+    properties:
+      baseCost:
+        type: number
+        description: The base cost
+      currencyType:
+        type: string
+        description: The currency type of the cost
+    example:
+      baseCost: 53.42
+      currencyType: 'US'
+responses: 
+  NotFound:
+    description: 404 - Not Found
+    headers:
+      Request-Id:
+        type: string
+    schema:
+      $ref: "#/definitions/Error"
+  BadRequest:
+    description: 400 - Bad Request
+    headers:
+      Request-Id:
+        type: string
+    schema:
+      $ref: "#/definitions/Error"
+  Forbidden:
+    description: 403 - Forbidden
+    headers:
+      Request-Id:
+        type: string
+    schema:
+      $ref: "#/definitions/Error"
+  Created:  
+    description: Created.
+    schema:
+      $ref: '#/definitions/BookingSession'
+  Gone:
+    description: Gone
+    schema:
+      $ref: '#/definitions/Error'
+  items: 
+    description: OK
+    schema:
+      $ref: '#/definitions/Item'
+  AvailableStore:
+    description: Okay - Available Store
+    schema:
+      type: array
+      items:
+        $ref: '#/definitions/AvailableStore'
+  Slot:
+    description: OK - Slots
+    schema:
+      type: array
+      items:
+        $ref: '#/definitions/Slot'
+  Appointment:
+    description: OK
+    schema:
+      type: array
+      items:
+        $ref: '#/definitions/Appointment'
+  OK:
+    description: OK
+    schema:
+      $ref: '#/definitions/Ok'
+
+    
+

```

Comment:
A bit confusing that the basepath is at the bottom

Reply:
I updated the basePath, changed the casing on the file names, and made all the response codes strings. Is there a way to reply to those other comments that aren't on a particular line?

Reply:
You just have to leave a comment on the first page, it's at the bottom